frontend/components/common/CompatibleDevice.tsx

```diff
@@ -26,102 +27,128 @@ export function CompatibleDevice({
-               </Text>
-               <Flex flexDir="column">
-                  {visibleVariants.map((variant) => (
-                     <Text
-                        key={variant}
-                        lineHeight="short"
-                        fontSize="xs"
-                        color="gray.600"
-                     >
-                        {variant}
-                     </Text>
-                  ))}
-               </Flex>
-               <Flex flexDir="column" {...getDisclosureProps()}>
-                  {hiddenVariants.map((variant) => (
-                     <Text
-                        key={variant}
-                        lineHeight="short"
-                        fontSize="xs"
-                        color="gray.600"
-                     >
-                        {variant}
-                     </Text>
-                  ))}
-               </Flex>
-               {truncateModels && hiddenVariants.length > 0 && (
+               </chakra.a>
+            </NextLink>
+            <Flex
+               flexDir="column"
+               {...(!truncateModels && {
+                  onClick: (event) => {
+                     event.preventDefault();
+                     event.stopPropagation();
+                  },
+               })}
+               sx={{
+                  'details .whenOpen': {
+                     display: 'none',
+                  },
+                  'details[open] .whenOpen': {
+                     display: 'initial',
+                  },
+                  'details[open] .whenClosed': {
+                     display: 'none',
+                  },
+               }}
+            >
+               {visibleVariants.map((variant) => (
                   <Text
-                     mb="2px"
+                     key={variant}
                      lineHeight="short"
                      fontSize="xs"
                      color="gray.600"
                   >
-                     And {hiddenVariants.length} other model
-                     {hiddenVariants.length > 1 ? 's' : ''}...
+                     {variant}
                   </Text>
-               )}
-               {!truncateModels && hiddenVariants.length > 0 && (
-                  <Text
-                     mt="2px"
-                     lineHeight="short"
-                     fontSize="xs"
-                     fontWeight="medium"
-                     color="brand.500"
-                     {...getButtonProps()}
-                     onClick={(event) => {
-                        onToggle();
-                        event.preventDefault();
-                        event.stopPropagation();
+               ))}
+            </Flex>
+            {truncateModels && hiddenVariants.length > 0 && (
+               <Text mb="2px" lineHeight="short" fontSize="xs" color="gray.600">
+                  And {hiddenVariants.length} other model
+                  {hiddenVariants.length > 1 ? 's' : ''}...
```

Comment:
Do you think this will be good enough?
```suggestion
                  And {hiddenVariants.length} more...
```

Reply:
Yea, probably good to be consistent in the wording anyway